gemfiles/rails_5.1.gemfile

```diff
@@ -15,9 +15,4 @@ gem "puma"
 gem "capybara", "~> 2.13"
 gem "selenium-webdriver"
 
-group :jekyll_plugins do
-  gem "jekyll-algolia"
-  gem "jekyll-redirect-from"
-end
-
```

Comment:
Removed these blocks as in other gemfiles. The `jekyll-algolia` gem requires ruby 2.3.0, and `graphql-ruby` still tests with 2.2.8, so it was producing errors from the auto-generated appraisals. I suspect somone had removed these from the earlier rails version by hand, and they aren't necessary for the tests anyway.